### ISSUE TITLE
oculus : eregos disapear

### ISSUE THREAD

**Issue by kokandz** [dalaranwow/dalaran-wow#5702]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: the oculus instance in norfendre : the last boss dragon eregos i think its named, the bug is : when someone dies and he go off combat, the dragon bug and disapear. 

**Expected behaviour**: i dont know what should happen but i guess he should at least reset to his previous position , not disapear 

**Steps to reproduce the problem**:

1. aggro the boss 
2. let the dragon mount you are in dies and jump to safety
3. use something like "invisibility , or cheat death " to go off combat , then the boss disapear

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
#2801
